### PR TITLE
Add support for audio

### LINKED ISSUE #158: Question: Add speed controler for <audio>?
Today I listening to a podcast online and wanted to increase the playback speed. It was fairly easy to do that after grabbing the `audio` element and setting it's playback rate.

```
var audio = document.getElementById("myAudio");
audio.playbackRate = 0.5;
```

My question is: would it make sense to open a PR & write the code for this? Or is this project focused on only on videos?

### PR DESCRIPTION
**Pull request by jacobcolbert**
Closes #158 
Based on #408 by @canarslan12.

### CODE REVIEW

File: inject.js [hunk 1]
```diff
@@ -371,7 +375,12 @@
       });
       observer.observe(document, { childList: true, subtree: true });
 
-      var videoTags = document.getElementsByTagName('video');
+      if (tc.settings.audioBoolean) {
+        var videoTags = document.querySelectorAll('video,audio');
```
Comment (igrigorik) [inline inject.js]: nitpicking, but since we're now picking up both audio and video, can we rename the var to `mediaTags`?

File: inject.js [hunk 2]
```diff
@@ -385,22 +394,22 @@
...
       var targetController = e.target.getRootNode().host;
-    }
+    } 
 
     videoTags.forEach(function(v) {
       var id = v.dataset['vscid'];
       var controller = document.querySelector(`div[data-vscid="${id}"]`);
-
-      // Don't change video speed if the video has a different controller
-      if (e && !(targetController == controller)) {
```
Comment (igrigorik) [inline inject.js]: This was just added. Are you intentionally removing it?

Comment (jacobcolbert) [reply inject.js]: No. I was trying to undo the addition of the code in my pull request #452

File: inject.js (hunk 1 above)
Comment (jacobcolbert) [reply inject.js]: It bugs me too. Do you want to change it everywhere? Lots of references to "video" in videospeedcontroller lol.

**Pull request merged**